realm/realm-library/src/main/cpp/io_realm_internal_TableQuery.cpp

```diff
@@ -19,9 +19,11 @@
 #include <shared_realm.hpp>
 #include <object_store.hpp>
 #include "util.hpp"
+#include "jni_util/java_local_ref.hpp"
 #include "io_realm_internal_TableQuery.h"
 
 using namespace realm;
+using namespace realm::jni_util;
```

Comment:
Well, my point is that you add a line `using namespace realm::jni_util;` to avoid writing `jni_util::` in front of the `JavaLocalRef`. It seems more trouble and less clear to my eye. So I would prefer we add `jni_util::`.
We don't have a standard for when to use `using` - but here and in the other files, it seems unnecessary and inconsistent as you in other files add `using` but still make calls with `jni_util::` :-)